src/main/kotlin/calculator/StringCalculator.kt

```diff
@@ -0,0 +1,38 @@
+package calculator
+
+class StringCalculator {
+    fun calculate(input: String?): Double {
+        val equation = validateEmpty(input)
+        var (result, operand, operator) = initialize()
+
+        val splitEquation = equation.split(" ")
+
+        for ((index, element) in splitEquation.withIndex()) {
+            if (isOperandIndex(index)) {
+                operand = element.toDoubleOrNull() ?: throw IllegalArgumentException("입력값이 잘못되었습니다.")
+                result = operator.operate(result, operand)
```

Comment:
`Triple`에 `operand`가 없어도 될 것 같아보이네용!
```suggestion
        var (result, operator) = initialize()

        val splitEquation = equation.split(" ")

        for ((index, element) in splitEquation.withIndex()) {
            if (isOperandIndex(index)) {
                val operand = element.toDoubleOrNull() ?: throw IllegalArgumentException("입력값이 잘못되었습니다.")
                result = operator.operate(result, operand)
```